docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -0,0 +1,152 @@
+
+It's highly likely that you have the two image files already in your add-in. They are specified in the [IconUrl](/javascript/api/manifest/iconurl) and [HighResolutionIconUrl](/javascript/api/manifest/highresolutioniconurl) elements in the XML manifest. But these are both optional elements, so it's possible your add-in project doesn't have one or both. These are required for an add-in that uses the unified manifest, so you need to create them if the project doesn't already have them. For guidance, see [Create an icon for your add-in](/office/dev/store/create-effective-office-store-listings#create-an-icon-for-your-add-in). 
+
+When you've added the files to the project, add **\<IconUrl\>** and **\<HighResolutionIconUrl\>** (in that order) to the XML manifest just below the **\<Description\>** element. The following is an example:
+
+```xml
+<OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="MailApp">
+  <Id>01234567-89ab-cdef-0123-4567-89abcdef0123</Id>
+  <Version>1.0</Version>
+  <ProviderName>Contoso</ProviderName>
+  <DefaultLocale>en-us</DefaultLocale>
+  <DisplayName DefaultValue="Great Add-in"/>
+  <Description DefaultValue="A great add-in."/>
+  <IconUrl DefaultValue="https://localhost:3000/assets/icon-64.png" />
+  <HighResolutionIconUrl DefaultValue="https://localhost:300/assets/icon-128.png" />
+
+  <!-- Other markup omitted -->
+```
+
+## Conversion tools and options
+
+There are several ways to carry out the remaining tasks, depending on what IDE and other tools you want to use for your project and on what tool was used to create the project. 
+
+- [Convert the project with Teams Toolkit](#convert-the-project-with-teams-toolkit)
+- [Projects created with the Office Yeoman Generator (aka "Yo Office")](#projects-created-with-the-office-yeoman-generator-aka-yo-office)
+- [NodeJS and npm projects that aren't created with Yeoman Generator](#nodejs-and-npm-projects-that-arent-created-with-yeoman-generator)
+
+### Convert the project with Teams Toolkit
+
+#### Prerequisites
+
+- Install [Visual Studio Code](https://code.visualstudio.com/).
+- Install [Teams Toolkit](/microsoftteams/platform/toolkit/install-teams-toolkit?tabs=vscode#install-teams-toolkit-for-visual-studio-code).
+
+#### Import the add-in project to Teams Toolkit
+
+1. Open Visual Studio Code and select Teams Toolkit icon in the **Activity Bar**.
+
+    :::image type="content" source="../images/teams-toolkit-icon.png" alt-text="Teams Toolkit icon.":::
+
+1. Select **Create a new app**.
+1. In the **New Project** drop down, select **Outlook add-in**.
+
+    :::image type="content" source="../images/teams-toolkit-create-outlook-add-in.png" alt-text="Screenshot showing four options in New Project drop down. The fourth option is called 'Outlook add-in'.":::
+
+1. In the **App Features Using an Outlook Add-in** drop down, select **Import an Existing Outlook Add-in**.
```

Comment:
Are these UI elements really Sentence Case? Seems like it should be "App features using an Outlook add-in." Please verify these strings match the UI.

Reply:
They match.